### PR TITLE
feat(auth): Added mock merit committee head login for preview environment

### LINKED ISSUE #61: Implement an additional provider in NextAuth so that google auth can be bypassed in preview deployments
We have an issue were preview deployments are useless as all features of the application are gated by Google Auth. Google Auth does not work in preview. So, implement a new simple Credentials Provider that makes auth for preview simple

Source: https://next-auth.js.org/deployment

### PR DESCRIPTION
**Pull request by Suraj-Ram**
Closes #61

## Description
* Added an additional mock login for a MERIT_COMMITTEE_HEAD user

## Things you especially want reviewed
* Check if it works

## Screenshots if Applicable
| Login button | Profile page |
| ------------- | ------------ |
| image | image |

## Checklist

- [x] Ticket number in PR title
- [x] Add ticket number to ("Closes #")
- [x] Move status to Code Review in GH Board
- [x] People added to reviewers
- [x] Asked for Review in Slack

### CODE REVIEW

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -33,6 +35,20 @@ const preview_providers = [
       };
     },
   }),
+  // Admin (committee head) Test User
+  CredentialsProvider({
+    name: 'MERIT_COMMITTEE_HEAD',
+    id: 'admin-test-provider',
+    // @ts-ignore comment
```
Comment (maxpinheiro) [inline src/pages/api/auth/[...nextauth].ts:42]: remove ts-ignore, to satisfy typescript:
- make id a string ('3')
- add another field to credentialsprovider, `credentials: {}`

**Review by maxpinheiro — CHANGES_REQUESTED** [review]
looks good, just one small change to satisfy typescript

File: src/pages/api/auth/[...nextauth].ts (hunk 1 above)
Comment (Suraj-Ram) [reply src/pages/api/auth/[...nextauth].ts]: Fixed, lmk if it works

**Review by maxpinheiro — APPROVED** [review]
LGTM 👍

**Pull request merged**